Source/Fuse.Alerts/Triggers.uno

```diff
@@ -0,0 +1,265 @@
+using Uno;
+using Uno.UX;
+
+using Fuse.Triggers.Actions;
+using Fuse.Scripting;
+
+namespace Fuse
+{
+
+	public class AlertArgs : EventArgs, IScriptEvent
+	{
+		string _buttonLabel;
+
+		public AlertArgs(string buttonLabel)
+		{
+			_buttonLabel = buttonLabel;
+		}
+		void IScriptEvent.Serialize(IEventSerializer s)
+		{
+			Serialize(s);
+		}
+
+		virtual void Serialize(IEventSerializer s)
+		{
+			s.AddString("buttonLabel", _buttonLabel);
+		}
+	}
+
+	public delegate void AlertHandler(object sender, AlertArgs args);
+
+	/**
+		This is trigger action for showing native alert dialog with a single button. Only available on iOS or Android
+
+		## Example
+
+			The following example shows how to use it:
+
+				<JavaScript>
+					module.exports = {
+						handler: function(data) {
+							if (data.buttonLabel == 'Yes'){
+								console.log("yes button clicked")
+							}
+						}
+					};
+				</JavaScript>
+				<Panel>
+					<Button Text="Display Alert" Alignment="Center">
+						<Clicked>
+							<ShowAlert Message="Hello world!" OkLabelButton="Yes" Handler="{handler}"/>
+						</Clicked>
+					</Button>
+				</Panel>
+	*/
+	public class ShowAlert: TriggerAction
+	{
+		Node _target;
+
+		string _title = "Alert";
+		/**
+		The title of the alert dialog
+		*/
+		public string Title
+		{
+			get
+			{
+				return _title;
+			}
+			set
+			{
+				_title = value;
+			}
+		}
+
+		/**
+		String message to show in the dialog
```

Comment:
Nit: Consider adding another tab inside doc-comments (multiple), for consistency.
```csharp
/**
    String message...
*/
```

Reply:
sorted